federation/backends/collections.md

```diff
@@ -1,6 +1,40 @@
 # Collections
 
-If back-ends offer/mirror the same datasets, it is required to align the metadata.
+If back-ends offer/mirror the same datasets, it is required to align names and metadata.
+
+The following standardized names have been agreed upon:
+
+- SENTINEL2_L1C
+- SENTINEL2_L2A
+- SENTINEL1_GRD
+- SENTINEL3_OLCI_L1B
+
+These are all Copernicus Missions, and the standard names refer to the archives prepared and distributed by ESA. When it is not possible/desirable to use this name as collection id, a 'common_name' can be added next to the 'id' property that identifies the collection as a standard archive.
+
 For the following datasets an agreement has been achieved:
```

Comment:
Maybe we can removed this and line 3/5 and put it into a general sentence at the beginning of the document. 

Something like "If back-ends offer/mirror the same datasets, it is required to align names and metadata. For the following collections and metadata an agreement has been achieved.

Reply:
ok, modified, please check again

Reply:
Looks good, thanks!